wealth-management-client/src/app/data/stocks-recommendations.json

```diff
@@ -0,0 +1,348 @@
+    "rcomDate": "18-Sep-2017",
+    "rcomTimeScale": "6",
+    "stoploss": 200.0,
+    "rcomBy": "StockAxis",
+    "script": "PARAGMILK"
+  },
+  {
+    "targetPrice": 219.0,
+    "rcomPrice": 145.0,
+    "rcomDate": "21-Nov-2017",
+    "rcomTimeScale": "6",
+    "stoploss": 116.0,
+    "rcomBy": "StockAxis",
+    "script": "PRAKASH"
+  },
+  {
+    "targetPrice": 712.0,
+    "rcomPrice": 480.0,
+    "rcomDate": "04-Dec-2017",
+    "rcomTimeScale": "6",
+    "stoploss": 380.0,
+    "rcomBy": "StockAxis",
+    "script": "PSPPROJECT"
+  },
+  {
+    "targetPrice": 1620.0,
+    "rcomPrice": 1080.0,
+    "rcomDate": "26-Dec-2017",
+    "rcomTimeScale": "6",
+    "stoploss": 865.0,
+    "rcomBy": "StockAxis",
+    "script": "RATNAMANI"
+  },
+  {
+    "targetPrice": 4125.0,
+    "rcomPrice": 2727.0,
+    "rcomDate": "21-Sep-2017",
+    "rcomTimeScale": "6",
+    "stoploss": 2200.0,
+    "rcomBy": "StockAxis",
+    "script": "SHARDA"
+  },
+  {
+    "targetPrice": 482.0,
+    "rcomPrice": 321.0,
+    "rcomDate": "19-Sep-2017",
+    "rcomTimeScale": "6",
+    "stoploss": 255.0,
+    "rcomBy": "StockAxis",
+    "script": "SURYAROSNI"
+  },
+  {
+    "targetPrice": 200.0,
+    "rcomPrice": 171.95,
+    "rcomDate": "10-Jul-2017",
+    "rcomTimeScale": "2",
+    "rcomBy": "Wealth",
+    "script": "BEL"
+  },
+  {
+    "targetPrice": 650.0,
+    "rcomPrice": 503.9,
+    "rcomDate": "13-Nov-2017",
+    "rcomTimeScale": "2",
+    "rcomBy": "Wealth",
+    "script": "BHARTIARTL"
+  },
+  {
+    "targetPrice": 3500.0,
+    "rcomPrice": 2405.6,
+    "rcomDate": "06-Nov-2017",
+    "rcomTimeScale": "2",
+    "rcomBy": "Wealth",
+    "script": "DRREDDY"
+  },
+  {
+    "targetPrice": 1123.0,
+    "rcomPrice": 829.55,
+    "rcomDate": "18-Dec-2017",
+    "rcomTimeScale": "2",
+    "rcomBy": "Wealth",
+    "script": "GUJGASLTD"
+  },
+  {
+    "targetPrice": 520.0,
+    "rcomPrice": 410.0,
+    "rcomDate": "14-Aug-2017",
+    "rcomTimeScale": "2",
+    "rcomBy": "Wealth",
+    "script": "IOC"
+  },
+  {
+    "targetPrice": 915.0,
+    "rcomPrice": 768.0,
+    "rcomDate": "17-Jul-2017",
+    "rcomTimeScale": "2",
+    "rcomBy": "Wealth",
+    "script": "LTTS"
+  },
+  {
+    "targetPrice": 330.0,
+    "rcomPrice": 248.9,
+    "rcomDate": "31-Jul-2017",
+    "rcomTimeScale": "2",
+    "rcomBy": "Wealth",
+    "script": "MAHINDCIE"
+  },
+  {
+    "targetPrice": 100.0,
+    "rcomPrice": 82.5,
+    "rcomDate": "25-Dec-2017",
+    "rcomTimeScale": "2",
+    "rcomBy": "Wealth",
+    "script": "NATIONALUM"
+  },
+  {
+    "targetPrice": 320.0,
+    "rcomPrice": 247.5,
+    "rcomDate": "11-Dec-2017",
+    "rcomTimeScale": "2",
+    "rcomBy": "Wealth",
+    "script": "PARAGMILK"
+  },
+  {
+    "targetPrice": 1750.0,
+    "rcomPrice": 1420.95,
+    "rcomDate": "30-Oct-2017",
+    "rcomTimeScale": "2",
+    "rcomBy": "Wealth",
+    "script": "PNBHOUSING"
+  },
+  {
+    "targetPrice": 220,
+    "rcomPrice": 157,
+    "rcomDate": "1-Jan-2010",
+    "rcomTimeScale": "4",
```

Comment:
We need to convert this into range, like 2-3 months, 3 to 6 months, 6 to 12 months and 12 to 18 months.